### DISCUSSION TITLE
Edit this border?

### DISCUSSION THREAD

**Discussion by maribox** [Alex313031/thorium#902] [Q&A]
![image](https://github.com/user-attachments/assets/e998cdec-173a-4171-ab1d-613f0d5c3946)

Hey, I was tinkering a little with themes and wondered whether you could style these 1px gray borders. 
They make the currently selected tab look like it's lower than the other tabs and annoy me. Is there any way to color them with themes or would I actually have to change code to get rid of these? Cheers!

**Comment by Alex313031** [comment]
@maribox You can disable them with the flag `chrome://flags/#force-disable-tab-outlines`

**Reply by maribox** [reply]
This is amazing thank you! But it only disables the tab outlines. Is there a way to disable the lower border too?
